The crates.io release does not include the benches or examples, but there is
still metadata related to them in Cargo.toml, clean it up so the autopkgtest
can run succesfully.

Index: textwrap/Cargo.toml
===================================================================
--- textwrap.orig/Cargo.toml
+++ textwrap/Cargo.toml
@@ -25,21 +25,6 @@ repository = "https://github.com/mgeisle
 [package.metadata.docs.rs]
 all-features = true
 
-[[example]]
-name = "termwidth"
-path = "examples/termwidth.rs"
-required-features = ["terminal_size"]
-
-[[bench]]
-name = "linear"
-path = "benches/linear.rs"
-harness = false
-
-[[bench]]
-name = "indent"
-path = "benches/indent.rs"
-harness = false
-
 [dependencies.smawk]
 version = "0.3.1"
 optional = true
@@ -55,11 +40,6 @@ optional = true
 [dependencies.unicode-width]
 version = "0.1.9"
 optional = true
-[dev-dependencies.criterion]
-version = "0.3.5"
-
-[dev-dependencies.lipsum]
-version = "0.8.0"
 
 [dev-dependencies.unic-emoji-char]
 version = "0.9.0"
